main.js

```diff
@@ -0,0 +1,366 @@
+//Hello, if you are reading this, I'm sorry. This code is a hot mess. Most of these functions are unnececarily long,
+// and could be split up to be much more concise and readable. But this is what I have as of 1am Monday morning,
+// and it works (mostly). I will take some time when I have it to make sure no one else has stress-dreams about it, I promise!
+
+
+/////// Start, Single Player, and Two Player Buttons, Status Bar///////
+
+let startButton = document.getElementById("start");
+startButton.addEventListener("click", startGame);
+
+let singlePlayer = document.getElementById("single-player");
+singlePlayer.checked = true;
+singlePlayer.addEventListener("click", check);
+
+let twoPlayer = document.getElementById("two-player");
+twoPlayer.addEventListener("click", check);
+
+let statusBar = document.getElementById("status-bar");
+
+document.getElementById("audio").volume = 0.1;
+
+//////Global variable initializations////////////
+
+let timer = setInterval(countTimer, 1000);
+let totalSeconds = 0;
+let gameOver = true;
+let turnNumber = 0;
+let playerOne = "";
+let playerTwo = "";
+let xSquares = [];
+let oSquares = [];
+let emptySquares = [1, 2, 3, 4, 5, 6, 7, 8, 9];
+let winCombos = [
+  [1, 2, 3],
+  [4, 5, 6],
+  [7, 8, 9],
+  [1, 4, 7],
+  [2, 5, 8],
+  [3, 6, 9],
+  [1, 5, 9],
+  [3, 5, 7]
+];
+
+//////Initializing game-board squares, should probably have made some loops but oh-well...////////
+
+let cell1 = document.getElementById("cell-1");
+let cell2 = document.getElementById("cell-2");
+let cell3 = document.getElementById("cell-3");
+let cell4 = document.getElementById("cell-4");
+let cell5 = document.getElementById("cell-5");
+let cell6 = document.getElementById("cell-6");
+let cell7 = document.getElementById("cell-7");
+let cell8 = document.getElementById("cell-8");
+let cell9 = document.getElementById("cell-9");
+
+cell1.squareNumber = 1;
+cell2.squareNumber = 2;
+cell3.squareNumber = 3;
+cell4.squareNumber = 4;
+cell5.squareNumber = 5;
+cell6.squareNumber = 6;
+cell7.squareNumber = 7;
+cell8.squareNumber = 8;
+cell9.squareNumber = 9;
+
+cell1.addEventListener("click", drawCell);
+cell2.addEventListener("click", drawCell);
+cell3.addEventListener("click", drawCell);
+cell4.addEventListener("click", drawCell);
+cell5.addEventListener("click", drawCell);
+cell6.addEventListener("click", drawCell);
+cell7.addEventListener("click", drawCell);
+cell8.addEventListener("click", drawCell);
+cell9.addEventListener("click", drawCell);
```

Comment:
Then here, you could use your NodeList or Array with a forEach() to set all of your event listeners